dcos_test_utils/dcos_api.py

```diff
@@ -422,9 +423,16 @@ def marathon(self):
     @property
     def metronome(self):
         new = self.copy()
-        new.default_url = self.default_url.copy(path='service/metronome/v1')
+        new.default_url = self.default_url.copy(path='service/metronome')
         return new
-
+    
+    @property
+    def jobs(self):
```

Comment:
This module effectively deprecates the metronome_one_off call below. At the very least, the guts of that method should be ripped out and replaced by synonymous calls to this class